dashboard/test/ui/features/pd/workshopCertificates.feature

```diff
@@ -3,6 +3,7 @@
 
 Feature: Basic appearance for Workshop Certificates
 
+@no_circle
```

Comment:
Disabling this scenario on Circle wasn't mentioned in your PR description.  What's the reason behind this?